app/src/main/java/org/luxoft/sdl_core/BluetoothHandler.java

```diff
@@ -219,7 +205,9 @@ public void OnLongMessageReceived(byte[] message) {
         mLongWriter.setCallback(new BluetoothLongWriter.LongWriterCallback() {
             @Override
             public void OnLongMessageReady(byte[] message) {
-                BluetoothGattCharacteristic responseCharacteristic = mPeripheral.getCharacteristic(SDL_TESTER_SERVICE_UUID, MOBILE_RESPONSE_CHARACTERISTIC);
+
+                BluetoothGattCharacteristic responseCharacteristic = mPeripheral.getCharacteristic(UUID.fromString(AndroidSettings.getStringValue(AndroidSettings.IniParams.SDL_TESTER_SERVICE_UUID))
+                        , UUID.fromString(AndroidSettings.getStringValue(AndroidSettings.IniParams.MOBILE_RESPONCE_CHARACTERISTIC)));
```

Comment:
@VadymLuchko please move out ini settings into a separate constants

Reply:
fixed in 2e09204